person.rb

```diff
@@ -16,5 +17,10 @@ def of_age?
   def can_use_services?
     @parent_permision || of_age?
   end
+
+  def validate_name
+    corrector = Corrector.new
+    @correct_name = corrector.correct_name(@name)
```

Comment:
- [ ] This should correct the `@name` instance variable already initialized in the **initialize** method, no need for `@correct_name`. Kindly fix this.

  You should have something like this:
  ```
  @name = @corrector.correct_name(@name)
  ```